src/scene_battle_rpg2k3.cpp

```diff
@@ -1217,6 +1221,16 @@ Scene_Battle_Rpg2k3::SceneActionReturn Scene_Battle_Rpg2k3::ProcessSceneActionFi
 						Main_Data::game_system->SePlay(Main_Data::game_system->GetSystemSE(Main_Data::game_system->SFX_Buzzer));
 					}
 					break;
+				case Win: // Win
+					for each (Game_Enemy* enemy in Main_Data::game_enemyparty->GetEnemies())
```

Comment:
I think you confused the syntax here (does this compile?)

Syntax should be:

```cpp
for (Game_Enemy* enemy: Main_Data::game_enemyparty->GetEnemies())
```

Reply:
It does, and I saw in your other review you found out it's a Microsoft compiler thing which predates the modern introduction of proper support in the language, it turns out it won't compile on Linux anyway. Oops! Fixed!